.github/workflows/iroha2-custom-image.yml

```diff
@@ -41,15 +42,19 @@ env:
 
 jobs:
   build_executor:
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, Linux, iroha2]
```

Comment:
It's not so heavy. May be on self-hosted it will be a bit faster, but we lose in time to rise up the runner and cost management